Library/Formula/subversion.rb

```diff
@@ -95,18 +95,6 @@ def install
       scons "install"
     end
 
-    if build.include? "unicode-path"
```

Comment:
I've killed this block because we're now up to Subversion 1.9, I'm likely going to remove the subversions in Versions relatively soon and to upgrade from 1.7 to the current formula you'd have had to not updated Homebrew for... 2 years, 6 months, 11 days. Seems like enough time for the notation to die.